Defs/Ammo/Advanced/12x20mmCharged.xml

```diff
@@ -0,0 +1,279 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+	<ThingCategoryDef>
+		<defName>Ammo12x20mmCharged</defName>
+		<label>12x20mm Charged</label>
+		<parent>AmmoAdvanced</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberCharge</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_12x20mmCharged</defName>
+		<label>12x20mm Charged</label>
+		<ammoTypes>
+			<Ammo_12x20mmCharged>Bullet_12x20mmCharged</Ammo_12x20mmCharged>
+			<Ammo_12x20mmCharged_AP>Bullet_12x20mmCharged_AP</Ammo_12x20mmCharged_AP>
+			<Ammo_12x20mmCharged_Ion>Bullet_12x20mmCharged_Ion</Ammo_12x20mmCharged_Ion>
+		</ammoTypes>
+		<similarTo>AmmoSet_ChargedPistol</similarTo>
+	</CombatExtended.AmmoSetDef>
+
+	<!-- ==================== Ammo ========================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" Name="12x20mmChargedBase" ParentName="SpacerSmallAmmoBase" Abstract="True">
+		<description>Charged shot ammo used by high caliber magnum charge weapons.</description>
```

Comment:
Should probably specify that it's intended for charge magnum *handguns*, so it's not mistaken at a glance for the charge equivalent of .338 lapua magnum, for instance.